Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,183 @@
+@import "color";
+$font-size:20px;
+body {
+  font-family: sans-serif;
+  .container-body {
+    background: linear-gradient(to left,rgb(163, 243, 226),rgb(45, 190, 190),rgb(2, 129, 129));
+    color: $white;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .calender { 
+      width: 480px;
+      background-color: $background-color;   
+      box-shadow: 5px 5px 10px $black;
+      @media screen and (max-width:768px) {
+        width: 320px;
+      }
+      .month {
+        height: 120px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        text-align: center;
+        padding: 0 20px;
+        border-bottom: 2px solid $aqua-green;
+        @media screen and (max-width:768px) {
+          height: 100px;
+        }
+        i {
+          font-size: $font-size;
+          cursor: pointer;
+          color: $aqua-green;
+        }
+        .month-name,
+        .current-year-name {
+          text-transform: uppercase;
+          letter-spacing: 2px;
+          font-size: 30px;
+          color: $aqua-green;
+          cursor: pointer;
+        }
+      }
+      .week-days {
+        display: flex;
+        padding: 20px 0px;
+      }
+      .week-days .week-day-name {
+        width: 60px;
+        text-align: center;
+      }
+      .days-container {
+        display: flex;
+        #week-number{
+          .week {
+            width: 60px;
+            height: 60px;
+            color: $light-blue;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            box-sizing: border-box;
+            @media screen and (max-width:768px) {
+              width: 40px;
+              height: 40px;
+            }
+          }
+        }
+      }
+      .days {
+        display: flex;
+        flex-wrap: wrap;
+        text-align: center;
+        width:420px;
+      }
+      .days .number-of-days {
+        width: 60px;
+        height: 60px;
+        display: flex;
+        justify-content: center;
+        align-items: center; 
+        text-shadow: 2px 2px 5px $text-shadow;
+        @media screen and (max-width:768px) {
+          width: 40px;
+          height: 40px;
+        }
+      }
+      .prev-days,.next-days {
+        opacity: 0.3;
+      }
+      .sunday {
+        color: $orange-red;
+      }
+      .event {
+        border-radius: 50%;
+        border: 2px solid $red;
+        box-sizing: border-box;
+        color: $red;
+      }
+      .today {
+        border: 2px solid $aqua-green;
+        box-sizing: border-box;
+      }
+      .days .number-of-days:hover:not(.week-number):not(.event){
+        box-sizing: border-box;
+        border: 2px solid $white;
+      }
+    }
+    .Scrollpanel {

```

Comment:
Class name should start with lower case and it should be hyphen separated if more than 1 word is used.

Reply:
Changed it to scroll-panel.